docs/admin/troubleshooting_updates.rst

```diff
@@ -107,13 +107,13 @@ Note that ``dom0`` and ``apply_dom0`` are separate steps.
    security-sensitive, and may requrie a reboot to take
    effect.
 
-``sd-*-template`` or ``whoix-gw-16`` update failures
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``sd-*-template`` or ``whoix-gateway-17`` update failures
```

Comment:
```suggestion
``sd-*-template`` or ``whonix-gateway-17`` update failures
```

Looks like we were/are missing an "n" in whonix here - unless that was intentional for some reason?

Reply:
thank you, fixed in [ec32763](https://github.com/freedomofpress/securedrop-workstation-docs/pull/180/commits/ec32763a40f2ce151c349771c4ecf1086f87d743)

Reply:
Perfect, thank you!